src/components/DetailItems/OwnerDetailItem.tsx

```diff
@@ -0,0 +1,35 @@
+import type { FC } from 'react';
+import { DetailsItem } from 'src/modules/Providers/utils/components/DetailsPage/DetailItem';
+import { OwnerReferencesItem } from 'src/modules/Providers/utils/components/DetailsPage/OwnerReferencesItem';
+import { useForkliftTranslation } from 'src/utils/i18n';
+
+import type { ResourceDetailsItemProps } from './utils/types';
+
+const OwnerDetailsItem: FC<ResourceDetailsItemProps> = ({
+  helpContent,
+  moreInfoLink,
+  resource,
+}) => {
+  const { t } = useForkliftTranslation();
+
+  const defaultMoreInfoLink =
+    'https://kubernetes.io/docs/concepts/overview/working-with-objects/owners-dependents/';
```

Comment:
Please create, if it does not exist, a central place for all links and move it there